1907-05-20.xml

```diff
@@ -2151,235 +2175,3387 @@
+                        possibly, have been more successful than others in training their Egyptian
+                        subordinates. Some, again, may be inclined to insist on a rather excessive
+                        standard of efficiency on the part of the Egyptian before they will readily
+                        acquiesce in foregoing the appoitment of an European. As Lord Cromer
+                        remarks, "I have never found on the part of any higher British officials in
+                        this country any tendency to question the wisdom of the policy, or the least
+                        reluctance to give effect to it when once they were convinced that a
+                        qualified Egyptian could be found to take any post which might happen to be
+                        vacant."</p>
+                    <p>The difficulties in the way of giving anything like immediate effect to the
+                        substitution on a large scale of Egyptian for European agency in the
+                        administrative work of the country are, in fact, very great. This matter is
+                        frequently discussed on the assumption that a number of places under
+                        Government are now occupied by Europeans for which competent Egyptians could
+                        without difficulty be found. This assumption certainly gives a very
+                        innocorrect view of the facts of the situation. There is no doubt that there
+                        are a few cases as to which it may be said that, if the European occupant of
+                        some post vacated his place, a competent Egyptian might at once be found to
+                        replace him. But, in the very large majority of cases, the reason why the
+                        European holds the post is that the supply of competent Egyptians is not
+                        nearly equal to the demand.</p>
+                </div>
+                <div type="advert" xml:id="deg-ad-nkh02">
+                    <head>NEW KHEDIVIAL HOTEL CAIRO.</head>
+                    <p>Samily Hotel. Built In 1904. Between Shepheard's Hotel and the Railway
+                        Station. Electric Light, Light.</p>
+                    <p>Sanitary arrangements up to-date. Open all the year round. Special terms for
+                        Government officials and Army of Occupation.Modern House. </p>
+                    <p><hi rend="bold">NEW MANAGMENT</hi></p>
+                    <p><measure type="indexNo">29242-26-11-90</measure></p>
+                </div>
+            </div>
+            <div type="page" n="3">
+                <div type="masthead">
+                    <head>3 THE EGYPTIAN GAZETTE, MONDAY MAY 20, 1907</head>
+                </div>
+                <div type="section" element="local">
+                    <head>LOCAL AND GENERAL.</head>
+                </div>
```

Comment:
move this `</div>` to the end of the section, so that all of the items are inside it.